eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/io/DbsnpJsonClusteredVariantsWriter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp2.io;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.data.mongodb.BulkOperationException;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import uk.ac.ebi.eva.accession.core.io.DbsnpClusteredVariantWriter;
+import uk.ac.ebi.eva.accession.core.listeners.ImportCounts;
+import uk.ac.ebi.eva.accession.core.persistence.DbsnpClusteredVariantEntity;
+
+import java.util.List;
+
+public class DbsnpJsonClusteredVariantsWriter implements ItemWriter<DbsnpClusteredVariantEntity> {
+
+    private static final Logger logger = LoggerFactory.getLogger(DbsnpJsonClusteredVariantsWriter.class);
+
+    private DbsnpClusteredVariantWriter dbsnpClusteredVariantWriter;
+
+    public DbsnpJsonClusteredVariantsWriter(MongoTemplate mongoTemplate, ImportCounts importCounts) {
+        dbsnpClusteredVariantWriter = new DbsnpClusteredVariantWriter(mongoTemplate, importCounts);
+    }
+
+    @Override
+    public void write(List<? extends DbsnpClusteredVariantEntity> clusteredVariants) {
+        try {
+            dbsnpClusteredVariantWriter.write(clusteredVariants);
+        } catch (BulkOperationException exception) {
```

Comment:
If we are going to wrap this class to add extra error notification, let's make it a bit more explicit. Instead of just showing a bunch of errors, listing duplicate RS IDs in a slightly more structured way would allow us to send that list to dbSNP so they can fix it in future releases.